modules/adding-a-tested-and-verified-runtime-for-the-multi-model-serving-platform.adoc

```diff
@@ -63,7 +63,7 @@ spec:
           tritonserver "--model-repository=/models/_triton_models" "--model-control-mode=explicit" "--strict-model-config=false" "--strict-readiness=false" "--allow-http=true" "--allow-grpc=true"  '
       command:
         - /bin/sh
-      image: nvcr.io/nvidia/tritonserver:23.05-py3
+      image: nvcr.io/nvidia/tritonserver@sha256:1bde45b946f50f3e23a562cc5401d5ae7af8cb4a2daf037a55548241fadbd6f0
```

Comment:
Isn't this an encryption key? Should this be in docs?

Reply:
@rpancham please advise.